style.css

```diff
@@ -1,166 +1,137 @@
-/* HEADER SECTION */
+/* --GENERAL SHAPE -- */
+body {
+  margin: 0;
+  width: 100%;
+}
 
-/* Title */
-#title {
-  color: #172b4d;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  padding-top: 20%;
-  margin-bottom: -10%;
+@font-face {
+  font-family: 'Inter-for-Microverse';
+  src: url('inter.ttf') format('truetype');
 }
 
-/* Subtitle */
-#title2 {
-  color: #36b37e;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  margin-bottom: 5%;
+/* --NAVIGATION MENU-- */
+
+.direction {
```

Comment:
- [ ] Kindly, make your navbar look as it look in the figma template to meet the requirement:

your design:
<img width="380" alt="image" src="https://user-images.githubusercontent.com/88894525/236651114-81ef90ff-98b1-46b4-b5d0-15d7c43c7717.png">

firgma design: 
<img width="337" alt="image" src="https://user-images.githubusercontent.com/88894525/236651089-f1209037-ddb6-429f-95f2-b8465835167a.png">